### ISSUE TITLE
Visual Studio reports `defaultSeverity` as warning

### ISSUE THREAD

**Issue by itanex** [Microsoft/tslint-microsoft-contrib#819]
### Bug Report
I am setting up TSLint support for Visual Studio 2017. I have included this plug-in in my tsconfig.json file and, while it takes a moment to run its analysis, I do get results in the Error Window. However, I set the `defaultSeverity` to `error` with the expectation that the TSLint results would show up as `Errors` in the Error Window. When I run TSLint through the CLI, the results are reported at Errors as I would expect. So I believe this is working. 

Is this the right expectation for Visual Studio?
Perhaps I am missing something in the setup for Visual Studio?

-   **`tslint-microsoft-contrib` version**: 0.3.1
-   **TSLint version**: 5.11.0
-   **TypeScript version**: 2.9.2
-   **Running TSLint via**: Visual Studio

#### tslint.json` configuration:

```json
{
  "defaultSeverity": "error",
  "extends": "tslint:recommended",
  "jsRules": {},
  "rules": {  },
  "rulesDirectory": []
}
```

#### Actual behavior
TSLint errors are displayed as `Warnings` in the Error Window

#### Expected behavior
TSLint errors are displayed as `Errors` in the Error Window

**Comment by IllusionMH** [comment]
@itanex this repo only provides lint rules and should not affect severity or integration with IDE.
As you noted - CLI gives correct results, so this issue should be filled against extension that you use in VS2017.
It may be the case that `defaultSeverity` is not yes supported by this extension.

**Comment by itanex** [comment]
Yeah I have been jumping around repos all morning and accidentally posted here.  I closed it since it was a mistake.
